q2_feature_table/plugin_setup.py

```diff
@@ -652,3 +654,59 @@
                 'splits are defined by values in metadata column.',
     examples={}
 )
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_feature_frequencies,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={},
+    outputs={'metadata': ImmutableMetadata},
+    input_descriptions={
+        'table': 'A table containing feature frequency information'
+    },
+    output_descriptions={
+        'metadata': 'Metadata generated from frequency data'
+    },
+    name='Tabulate feature frequencies',
+    description='Tabulates feature frequencies from a feature table '
+                'and generates metadata',
+    examples={}
+)
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_sample_frequencies,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={},
+    outputs={'metadata': ImmutableMetadata},
+    input_descriptions={
+        'table': 'A table containing sample frequency information'
+    },
+    output_descriptions={
+        'metadata': 'Metadata generated from frequency data'
+    },
+    name='Tabulate sample frequencies',
+    description='Tabulates feature frequencies from a feature table '
+                'and generates metadata',
+    examples={}
+)
+
+plugin.pipelines.register_function(
+    function=q2_feature_table.summarize_plus,
```

Comment:
As it stands, I think this will fail (or should at least) on the `FeatureTable` types other than `Frequency`, since the `Pipeline` allows all three as input and the two new actions only allow `Frequency` as input.